bascule/basculehttp/log.go

```diff
@@ -0,0 +1,34 @@
+package basculehttp
+
+import (
+	"context"
+	"os"
+
+	"github.com/go-kit/kit/log"
+)
+
+var (
+	defaultLogger = log.NewJSONLogger(log.NewSyncWriter(os.Stdout))
+
+	callerKey    interface{} = "caller"
+	messageKey   interface{} = "msg"
+	errorKey     interface{} = "error"
+	timestampKey interface{} = "ts"
+)
+
+// logger we expect for the decorators
+type Logger interface {
+	Log(keyvals ...interface{}) error
+}
+
+type DefaultLogger struct {
```

Comment:
There's no need for a wrapper `struct`.  You can just use your `bascule.Logger` directly.